opm/simulators/flow/GenericThresholdPressure_impl.hpp

```diff
@@ -227,6 +227,12 @@ getRestartVector() const
         return {};
 
     std::vector<Scalar> result(numEquilRegions_ * numEquilRegions_, 0.0);
+
+    if (this->restart_) {
```

Comment:
This certainly looks correct to me, even to the point where always returning `this->thpres_ ` should be correct, also for non-restart cases? After all, that array is initialized by code identical to the block below?

Reply:
That makes sense! Updating..

Reply:
Needed a small fix for parallel without restart, but seems to work.